bin/init.bash

```diff
@@ -99,7 +108,19 @@ generate_base_wc_config() {
     if [[ -f $file ]]; then
         yq merge "$tmpfile" "$file" --inplace -a=overwrite --overwrite --prettyPrint
     fi
+
+    cloud_provider=$(yq r -e "$tmpfile" 'global.cloudProvider')
+    storageClass=$(yq r -e "$tmpfile" 'global.storageClass')
+    if [ "$cloud_provider" == "exoscale" &&  "storageClass" == "nfs" ]
```

Comment:
I see that this check is different than for service cluster - should it?
Would it be possible to create a function for setting the nfs ip instead, and call it for both sc and wc? :slightly_smiling_face: